Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.1.0"
 [deps]
 BlockDiagonals = "0a1fb500-61f7-11e9-3c65-f5ef3456f9f0"
 IterativeSolvers = "42fd0dbc-a981-5370-80f2-aaf504508153"
+JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
```

Comment:
Do we need JuMP in main dependencies? Shouldn't it be a Test (and possibly docs) dep?